kbase-extension/static/kbase/js/widgets/narrative_core/upload/fileUploadWidget.js

```diff
@@ -122,32 +126,41 @@ define([
                     $('#clear-all-btn-container').remove();
                     $('#clear-all-btn').remove();
                     $dropzoneElem.find('#global-info').css({'display': 'none'});
-                    $($dropzoneElem.find('#total-progress .progress-bar')).css({'width': '0%'});
+                    $($dropzoneElem.find('#total-progress .progress-bar')).css({'width': '0'});
                 })
-                .on('error', (err) => {
+                .on('error', (erroredFile) => {
+                    var $errorElem = $(erroredFile.previewElement);
+                    $errorElem.find('#upload_progress_and_cancel').hide();
+                    $errorElem.find('#dz_file_row_1').css({'display': 'flex', 'align-items': 'center'});
+                    $errorElem.css('color', '#DF0002');
+                    $errorElem.find('#error_icon').css('display', 'flex');
+                    this.removeProgressBar($dropzoneElem);
+
+                    // Set error message
                     let errorText = 'unable to upload file!';
-                    if (err && err.xhr && err.xhr.responseText) {
-                        errorText = err.xhr.responseText;
+                    var $errorMessage = $errorElem.find('#error_message');
+
+                    // I don't know how to determine if the file was too big other than looking at the preview message
```

Comment:
yeah looks like dropzone doesn't offer much help here, there's a maxfilesexceeded event but not for maxFilesize 
https://www.dropzonejs.com/#events 

this approach seems to make sense given that

Reply:
The `erroredFile` is a standard WebAPI [File](https://developer.mozilla.org/en-US/docs/Web/API/File) object, so it has a size attribute which gives up the size in bytes. In theory, that should be the error triggered by the `maxFilesize` option way up above where the Dropzone is created. Maybe that's useful?